diff -Naur a/tools/build/create-jvm-runner.pl b/tools/build/create-jvm-runner.pl
--- a/tools/build/create-jvm-runner.pl	2015-12-25 20:23:19.000000000 +0100
+++ b/tools/build/create-jvm-runner.pl	2016-05-11 18:25:15.282783853 +0200
@@ -64,6 +64,8 @@
           . ' -cp ' . ($^O eq 'MSWin32' ? '"%CLASSPATH%";' : '$CLASSPATH:') . $classpath
           . ' -Dperl6.prefix=' . $prefix
           . ' -Djna.library.path=' . $sharedir
+          . ' -Djava.library.path=' . $ENV{'RAKUDO_LIBJNIDISPATCH_DIR'}
           . ($^O eq 'MSWin32' ? ' -Dperl6.execname="%~dpf0"' : ' -Dperl6.execname="$0"');
+printf("\n\nLIBJNIDISPATCH: $ENV{'RAKUDO_LIBJNIDISPATCH_DIR'}\n\n");

 if ($debugger) {
